packages/client/src/redux/User/action.ts

```diff
@@ -36,6 +43,11 @@ export const signup = (userData: CreateUserDTO): AppThunk => {
             .catch((err: AxiosError) => {
                 if (err.response) {
                     dispatch(setError(err.response.data.errors));
+                    dispatch(
```

Comment:
Do you want to toast for error as well or are you notifying the error in the Signup box directly?

Reply:
Thinking of adding one since my switch statement only catches certain error (wrong password/invalid fields)

Reply:
Hahaha sounds good. I'm also gonna be going around adding some toasts as well.
Currently thinking of one each when login, signup, logout succeeds.